src/rules.js

```diff
@@ -2687,5 +2687,72 @@ module.exports = [
         "arguments": [{
             "a": "auto"
         }]
+    },
+    /**
+    ==================================================================
+    SVG
+    ==================================================================
+    */
+    {
+        "type": "pattern",
+        "name": "fill",
```

Comment:
The 'name' field is used when we generate the reference docs, so it needs to be formatted.  I'd go with something like "Fill (SVG)", etc for these